tests/scripts/all.sh

```diff
@@ -1510,6 +1510,28 @@ component_test_sw_inet_pton () {
     make test
 }
 
+# Disabling of key enrollment function in PK module is experimental, so albeit
```

Comment:
Thanks for trying it out, I generally insist on automated tests, but in this instance, I think this is overkill. In particular, I'm not a big fan of having to mark a bunch of cases as depending on `MBEDTLS_PSA_CRYPTO_CLIENT` as this might send the wrong message.

I think for the level of support we want now (which is no official support), it's better not to test this.

Reply:
I understand :)
Side question: what about the fixes I did in the tests? Without anything that can test those `depends_on` additions it's hard to provide a clear justification for them. I would say that also the fixes I did for the tests are to be removed. Wdyt?

Reply:
Yes, actually removing those is the primary reason I'd like to remove this.